src/assets/css/main.css

```diff
@@ -362,7 +362,7 @@ p.featured-content {
 }
 blockquote {
   border: 0;
-  font-family: 'Gotham';
+  font-family: 'Gotham' sans-serif;
```

Comment:
Does this work without the comma separating the values?

Reply:
Does need to be comma separated. I checked in the Chrome debug tool.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family